pkg/cli/credential_login.go

```diff
@@ -30,6 +30,7 @@ acorn login ghcr.io`,
 }
 
 type CredentialLogin struct {
+	NoValidate    bool   `usage:"Bypass login validation" short:"v"`
```

Comment:
lets just not have a short version. There are better uses for `-v`

Reply:
Also, after seeing it, I think i like `--skip-validate` better than `--no-validate`